### PR TITLE
Plain javascript

### PR DESCRIPTION
**Pull request by billionsjoel**
**This PR creates an awesome book app.**

- Create a collection that keeps a list of books 
- A function to add a new book to the collection.
- A function to remove a book from the collection using array filter
- Display all books saved in the collection
- preserve data on local storage.
- Add and remove books buttons.

### CODE REVIEW

File: assets/js/hardworker.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+  }
+
+  bookList.innerHTML = booksBuild;
+}
+
+function removeBook(bookId) {
+  booksData = booksData.filter((books) => books.id !== bookId);
+  localStorage.setItem('bookData', JSON.stringify(booksData));
+  readBookData();
+}
```
Comment (elyor-doniyorov) [inline assets/js/hardworker.js:66]: - [x] Make sure that when a user clicks the "Remove" button:
- The correct book is removed from the collection.
- The correct book disappears from the page.
Kindly fix it 🙏

File: assets/js/hardworker.js [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+const localStorageData = JSON.parse(localStorage.getItem('bookData'));
+function getLocalStorageData() {
+  if (localStorageData !== null) {
+    booksData = localStorageData;
+    readBookData();
+  }
+}
+
+window.onresize = getLocalStorageData();
```
Comment (elyor-doniyorov) [inline assets/js/hardworker.js:77]: - [x] When reloading the page the I could not see the books. Make sure data is preserved in the browser's memory by using [localStorage](https://github.com/microverseinc/curriculum-html-css/blob/main/local_storage.md), reload the page manually to check this after adding a book and removing a book😃

**Review by elyor-doniyorov — CHANGES_REQUESTED** [review]
Hi @billionsjoel and @clintonjosephs, 

Good job so far!

## To Highlight 👏 
- It has a basic UI implemented with plain HTML;
- It displays all books saved in the collection in the top part of the page;
- When a user clicks the "Add" button a new book is added to the collection;

There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: assets/js/hardworker.js (hunk 1 above)
Comment (clintonjosephs) [reply assets/js/hardworker.js:66]: Thank you for pointing this out, it was a slight oversight. It works correctly now

File: assets/js/hardworker.js (hunk 2 above)
Comment (clintonjosephs) [reply assets/js/hardworker.js:77]: Thank you for pointing this out, it was a slight oversight, work fine now

**Review by dasileker — APPROVED** [review]
## **Hi @billionsjoel,** 

## **Congratulations! 🎉 Project Approved ✅**
Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀 
 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**